ProiectPAO/src/com/company/services/WriteToAudit.java

```diff
@@ -0,0 +1,38 @@
+package com.company.services;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.time.LocalTime;
+import java.util.Date;
+
+public class WriteToAudit {
+    private static WriteToAudit instance;
```

Comment:
private constructor missing, same as in ReadFromFile - also, instead of creating new objects by calling constructor, you should call getInstance methods in Main